tdconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
```

Comment:
tsconfig?

Reply:
This is for TypeDoc, which has its own parallel config file.